agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/agogosml/Dockerfile.agogosml

```diff
@@ -24,7 +24,7 @@ FROM base-deps as agogosml
 
 ENV PYTHONUNBUFFERED=1
 
-RUN pip install agogosml
+RUN pip install agogosml=={{cookiecutter.AGOGOSML_CLI_VERSION_MAJOR}}.{{cookiecutter.AGOGOSML_CLI_VERSION_MINOR}}
```

Comment:
I assume that you want this command to install version `x.y.{whatever}`, right? If so, I don't think this command works. I just tested this:

```sh
python3 -m pip install --user flake8==3.7
```

This installed `flake8==3.7.0` although `3.7.7` exists.

Reply:
Yep; I originally had `pip install ~={{cookiecutter.AGOGOSML_CLI_VERSION_MAJOR}}.{{cookiecutter.AGOGOSML_CLI_VERSION_MINOR}}` but misunderstood previous comments so back to the other thread.